cmd/kola/options.go

```diff
@@ -215,5 +215,11 @@ func syncCosaOptions() error {
 		}
 	}
 
+	if kola.Options.IgnitionVersion == "" {
```

Comment:
With how the code is currently laid out `kola.Options.IgnitionVersion` will never be an empty string when entering this function. It's auto-set to something in [`syncOptions`](https://github.com/coreos/mantle/blob/master/cmd/kola/options.go#L193-L199) based on the distribution & [`syncCosaOptions` is only called after `syncOptions`](https://github.com/coreos/mantle/blob/master/cmd/kola/kola.go#L119-L122). Should this PR be on top of #1188?

Reply:
Yes, it is intended to be on top of #1188 - I split it out for clarity.  But I've now merged #1188 so this code is now correct, right?